docs/sphinx/user_guide/feature/policies.rst

```diff
@@ -275,156 +275,218 @@ RAJA policies for GPU execution using CUDA or HIP are essentially identical.
+ cuda/hip_thread_syncable_loop<dims...>    kernel (For)  Similar to thread-loop
+                                           launch (loop) policy, but safe to use
+                                                         with Cuda/HipSyncThreads
+ cuda/hip_thread_size_x_direct<size>       kernel (For)  Same as thread_x_direct
+                                           launch (loop) policy above but with
+                                                         a compile time number of
+                                                         threads
+ cuda/hip_thread_size_y_direct<size>       kernel (For)  Same as above, but map
+                                           launch (loop) to threads in y-dim
+ cuda/hip_thread_size_z_direct<size>       kernel (For)  Same as above, but map
+                                           launch (loop) to threads in z-dim
+ cuda/hip_flatten_threads_{xyz}_direct     launch (loop) Reshapes threads in a
+                                                         multi-dimensional thread
+                                                         team into one-dimension,
+                                                         accepts any permutation
+                                                         of dimensions
+                                                         (expt namespace)
+ cuda/hip_block_x_direct                   kernel (For)  Map loop iterates
+                                           launch (loop) directly to GPU thread
+                                                         blocks in x-dimension,
+                                                         one iterate per block
+ cuda/hip_block_y_direct                   kernel (For)  Same as above, but map
+                                           launch (loop) to blocks in y-dimension
+ cuda/hip_block_z_direct                   kernel (For)  Same as above, but map
+                                           launch (loop) to blocks in z-dimension
+ cuda/hip_block_x_loop                     kernel (For)  Similar to
+                                           launch (loop) block-x-direct policy,
+                                                         but use a grid-stride
+                                                         loop.
+ cuda/hip_block_y_loop                     kernel (For)  Same as above, but use
+                                           launch (loop) blocks in y-dimension
+ cuda/hip_block_z_loop                     kernel (For)  Same as above, but use
+                                           launch (loop) blocks in z-dimension
+ cuda/hip_block_size_x_direct<size>        kernel (For)  Same as block_x_direct
+                                           launch (loop) policy above but with
+                                                         a compile time number of
+                                                         blocks
```

Comment:
I find the 'size' parameter a little confusing given that the type name contains 'block_size'. Is it clearer to do this? '*_block_size_*direct<nblocks> ?

Reply:
I changed it to block_size, maybe x_block_size or nxthreads or something would be better?

Reply:
I changed ti to nxthreads and nxblocks to differentiate.